source-code/website/src/pages/marketplace/index.page.tsx

```diff
@@ -0,0 +1,304 @@
+import { Layout } from "../Layout.jsx"
+import { Meta, Title } from "@solidjs/meta"
+import { marketplaceItems as items } from "@inlang/marketplace/src"
+import { For, Show, Accessor, createSignal, createEffect } from "solid-js"
+import { Chip } from "#src/components/Chip.jsx"
+import { SearchIcon } from "../editor/@host/@owner/@repository/components/SearchInput.jsx"
+import { Button } from "../index/components/Button.jsx"
+import { GetHelp } from "#src/components/GetHelp.jsx"
+import Plus from "~icons/material-symbols/add-rounded"
+import Package from "~icons/material-symbols/package-2"
+
+export function Page() {
+	const [textValue, setTextValue] = createSignal<string>("")
+	const [tags, setTags] = createSignal<Record<string, any>[]>([])
+	const [results, setResults] = createSignal<number>(0)
+
+	createEffect(() => {
+		items.forEach((item: any) => {
+			setTags((tags) => {
+				const newTags = [...tags]
+				const tagId = item.id.split(".")[1].toLowerCase()
+
+				const tagIndex = newTags.findIndex((tag) => tag.name === tagId)
+
+				if (tagIndex === -1) {
+					newTags.push({ name: tagId, activated: true })
+				}
+
+				return newTags
+			})
+		})
+	})
```

Comment:
Why are you using `createEffect`? 

The marketplace items seem static. You can directly derive the tags: 

```ts

const tags = items.map((item) => {
  // ... 
})

```

Reply:
@floriandwt seems to still not be resolved.